### ISSUE TITLE
Provider package via bower

### ISSUE THREAD

**Issue by CaryLandholt** [fulup-bzh/DatePicker#6]
It would be nice to pull down the DatePicker via bower.

http://bower.io/docs/creating-packages/

**Comment by fulup-bzh** [comment]
Yes, I use NPM with few other packages, but never took the time to build anything with bower. If you have the time to make it. Feel free to push bower json file onto git and upload DatePicker on bower.

**Comment by pixels4nickels** [comment]
Yes, please. This is a huge drawback when using in a serious project where maintaining depends is key.

**Comment by fulup-bzh** [comment]
Please patch, I would be more than happy to merge your Bower config
